sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/aio/_document_model_administration_client_async.py

```diff
@@ -461,11 +461,11 @@ def list_operations(self, **kwargs: Any) -> AsyncItemPaged[OperationSummary]:
 
     @distributed_trace_async
     async def get_operation(self, operation_id: str, **kwargs: Any) -> OperationDetails:
-        """Get a document model operation by its ID.
+        """Get a model operation by its ID.
```

Comment:
just wondering, if everything is a document model, why is this one just a model operation?

Reply:
I was actually thinking of removing the word model completely, since this endpoint is about operations. After checking the swagger description for this endpoint, I am going to remove model from this description as well since the service is leaving this open to other types of operations, not just limited to document models. Fixing!

Reply:
Done